sdk/src/assertions/bmff_hash.rs

```diff
@@ -985,10 +1007,10 @@ impl BmffHash {
         asset_path: &std::path::Path,
         fragment_paths: &Vec<std::path::PathBuf>,
         output_dir: &std::path::Path,
-        local_id: u32,
-        unique_id: Option<u32>,
+        local_id: usize,
+        unique_id: Option<usize>,
     ) -> crate::Result<()> {
-        let max_proofs: usize = 4; // todo: calculate (number of hashes to perform vs size of manifest) or allow to be set
+        let max_proofs = get_settings_value::<usize>("core.merkle_tree_max_proofs").unwrap_or(5);
```

Comment:
Can we default this in settings instead?

Reply:
We do default it in settings.

Reply:
Making it an errror if not set.